crypto/kzg/src/lib.rs

```diff
@@ -36,20 +45,31 @@ impl From<c_kzg::Error> for Error {
 #[derive(Debug)]
 pub struct Kzg {
     trusted_setup: KzgSettings,
+    context: PeerDASContext,
 }
 
 #[automock]
 impl Kzg {
     /// Load the kzg trusted setup parameters from a vec of G1 and G2 points.
     pub fn new_from_trusted_setup(trusted_setup: TrustedSetup) -> Result<Self, Error> {
+        // Initialize the trusted setup using default parameters
+        //
+        // Note: One can also use `from_json` to initialize it from the consensus-specs
+        // json string.
+        let peerdas_trusted_setup = PeerDASTrustedSetup::default();
+        // Set the number of threads to be used
+        //
+        // we set it to 1 to match the c-kzg performance
+        const NUM_THREADS: usize = 1;
```

Comment:
What threading does this constant activate? We are already parallelizing on a per-blob basis, do we need more?

Reply:
This is threading internal to the library, ie there are parts that you can multithread inside of the library when computing proofs for example. You can leave this at 1, if you think its not beneficial